### DISCUSSION TITLE
GitHub is not a social network

### DISCUSSION THREAD

**Discussion by JCKodel** [community/community#128275] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why in the hell we're seeing "people following me" and all that crap from social media, while our recent activities is capped at bottom left and we can't have really useful links to our shite?

**Comment by ALICIALVO** [comment]
it is a social network but for a different category and not like instagram/tiktok etc..
but noting is perfect there is always a place for change and improvement(:

**Comment by asalt** [comment]
I was skeptical at first as well, but I am actually finding use in seeing what code people I'm following contribute to / fork / star.
